[RefGuide] Various Lucene/Solr, Solr/Lucene cleanups (#799)
apache/solr: solr/solr-ref-guide/modules/query-guide/pages/dismax-query-parser.adoc

=== mm (Minimum Should Match) Parameter

> When processing queries, Solr recognizes three types of clauses: mandatory, prohibited, and "optional" (also known as "should" clauses).

Author: From #800
```suggestion
When processing queries, there are three types of clauses: mandatory, prohibited, and "optional" (also known as "should" clauses).
```
